Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -551,7 +551,20 @@ def install
 
   def test_home
     assert_equal HOMEBREW_WWW,
-                 cmd("home", {"HOMEBREW_BROWSER" => "echo"})
+                cmd("home", {"HOMEBREW_BROWSER" => "echo"})
+                 
```

Comment:
Please try not to change the indentation of existing code. Also notice there's trailing whitespace in this line (though GitHub sadly doesn't highlight it). Most programming editors can be configured to automatically trim trailing whitespace on save, something I can highly recommend. 😸